bundle/manifests/gpu-operator-certified.clusterserviceversion.yaml

```diff
@@ -189,7 +189,7 @@ metadata:
     capabilities: Deep Insights
     categories: AI/Machine Learning, OpenShift Optional
     certified: "true"
-    containerImage: registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator:main-latest-ubi8
+    containerImage: registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator:release-24.6-latest-ubi8
```

Comment:
@cdesiniotis the CI is failing with:
```
level=error msg="failed to validate images: failed to validate image gpu-operator-image: failed to get image manifest: failed to get manifest registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator:release-24.6-latest-ubi8: request failed: not found [http 404]: {\"errors\":[{\"code\":\"MANIFEST_UNKNOWN\",\"message\":\"manifest unknown\",\"detail\":{\"Tag\":\"release-24.6-latest-ubi8\"}}]}\n"
```
Are these images used, or should we be using the `ghcr.io` images?

Reply:
makes sense to publish latest images on release branches to ghcr.io. The image in gitlab exists though: [registry.gitlab.com/nvidia/kubernetes/gpu-operator/staging/gpu-operator:release-24.6-latest-ubi8 ](https://gitlab.com/nvidia/kubernetes/gpu-operator/-/jobs/7540303872)